src/auxinfo/auxinfo_commit.rs

```diff
@@ -123,6 +125,9 @@ impl CommitmentScheme {
         // Public parameters in this decommit must be consistent with each other
         scheme.clone().public_key.verify(context)?;
 
+        let lower_bound = BigNumber::one() << ELL;
```

Comment:
issue: `ELL` is not the correct bound for the size of the Paillier modulus. Please review the documentation for the constants in `parameters.rs`, or check our internal confluence document called "Parameter selection".